solidity/migrations/2_deploy_contracts.js

```diff
@@ -80,6 +82,27 @@ module.exports = async function(deployer, network) {
   await deployer.link(Reimbursements, KeepRandomBeaconOperator);
   await deployer.link(BLS, KeepRandomBeaconOperator);
 
+  if(network === 'local') {
+    const initializationPeriod = 120;
```

Comment:
`initializationPeriod` is already declared above, can be just overwritten

Reply:
It has to stay at `1`. If we do it temporarily or committed by accident, that's fine but before we merge it to `master` we need to make sure it's `1`.